$OpenBSD: patch-dodo_py,v 1.10 2015/05/18 23:58:26 bcallah Exp $

glib-2.0.vapi:967.53-967.68: error: Access to instance member `length' from nullable reference denied

--- dodo.py.orig	Sat May 16 04:11:39 2015
+++ dodo.py	Mon May 18 18:34:06 2015
@@ -94,13 +94,12 @@ def task_pkg_flags():
 
 valac_options = [
 	'--thread',
-	'--enable-experimental-non-null',
 	'--enable-experimental',
 	'--target-glib=2.34', # see bug 0000004
 	'--define=LINUX'
 	]
 
-libbirdxml = Vala(src='libbirdxml', build='build', library='birdxml', so_version=version.LIBBIRDXML_SO_VERSION, pkg_libs=LIBBIRD_XML_LIBS)
+libbirdxml = Vala(src='libbirdxml', build='build', library='birdxml', so_version='${LIBbirdxml_VERSION}', pkg_libs=LIBBIRD_XML_LIBS)
 def task_libbirdxml():
 
     if config.POSIXVALA == True:
@@ -113,7 +112,7 @@ def task_libbirdxml():
     yield libbirdxml.gen_ln()
     
 
-libbirdgems = Vala(src='libbirdgems', build='build', library='birdgems', so_version=version.LIBBIRDGEMS_SO_VERSION, pkg_libs=LIBBIRD_LIBS, vala_deps=[])
+libbirdgems = Vala(src='libbirdgems', build='build', library='birdgems', so_version='${LIBbirdgems_VERSION}', pkg_libs=LIBBIRD_LIBS, vala_deps=[])
 def task_libbirdgems():
     yield libbirdgems.gen_c(valac_options)
     yield libbirdgems.gen_o(['-fPIC'])
@@ -121,7 +120,7 @@ def task_libbirdgems():
     yield libbirdgems.gen_ln()
 
 
-libbird = Vala(src='libbirdfont', build='build', library='birdfont', so_version=version.SO_VERSION, pkg_libs=LIBS, vala_deps=[libbirdgems, libbirdxml])
+libbird = Vala(src='libbirdfont', build='build', library='birdfont', so_version='${LIBbirdfont_VERSION}', pkg_libs=LIBS, vala_deps=[libbirdgems, libbirdxml])
 def task_libbirdfont():
     yield libbird.gen_c(valac_options)
     yield libbird.gen_o(['-fPIC -I./build/', """-D 'GETTEXT_PACKAGE="birdfont"'"""])
